### PR TITLE
New EasyConfigs for FreeXL with various toolchains

### PR DESCRIPTION
**Pull request by valtandor**
One of these is required as a dependency for GRASS 7.0.2. The others were installed locally for other purposes. Note that a CMake dependency exists; a CMake EasyConfig has been included to make testing easier.

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
@valtandor: needs sync with `develop` now that #2408 is merged

**Comment by valtandor** [comment]
Synced with develop.

File: easybuild/easyconfigs/f/FreeXL/FreeXL-1.0.1-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+easyblock = 'ConfigureMake'
+
+homepage = "https://www.gaia-gis.it/fossil/freexl/index"
+description = "FreeXL is an open source library to extract valid data from within an Excel (.xls) spreadsheet."
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'usempi': False, 'pic': True}
+
+download_suffix = ''
```
Comment (boegel) [inline easybuild/easyconfigs/f/FreeXL/FreeXL-1.0.1-intel-2015a.eb]: please remove?

File: easybuild/easyconfigs/f/FreeXL/FreeXL-1.0.0g-goolf-1.5.14.eb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+easyblock = 'ConfigureMake'
+
+homepage = "https://www.gaia-gis.it/fossil/freexl/index"
+description = "FreeXL is an open source library to extract valid data from within an Excel (.xls) spreadsheet."
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+toolchainopts = {'usempi': False, 'pic': True}
+
+download_suffix = ''
```
Comment (boegel) [inline easybuild/easyconfigs/f/FreeXL/FreeXL-1.0.0g-goolf-1.5.14.eb]: this is useless, so please remove?

File: easybuild/easyconfigs/f/FreeXL/FreeXL-1.0.2-foss-2015a.eb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+easyblock = 'ConfigureMake'
+
+homepage = "https://www.gaia-gis.it/fossil/freexl/index"
+description = "FreeXL is an open source library to extract valid data from within an Excel (.xls) spreadsheet."
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+toolchainopts = {'usempi': False, 'pic': True}
+
+download_suffix = ''
```
Comment (boegel) [inline easybuild/easyconfigs/f/FreeXL/FreeXL-1.0.2-foss-2015a.eb]: please remove?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 3 (3 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/08f15196131e3b186fab for a full test report.

**Comment by boegel** [comment]
@valtandor: is there an alternative download URL for older versions? tarballs for older versions are no longer there...

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 3 (3 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/3458c32be08ed750b946 for a full test report.

**Comment by valtandor** [comment]
@boegel: `download_suffix` has now been removed, and the download URL updated, in existing EasyConfigs as well as the new one I supplied.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/4772bc576c1ff3f13303 for a full test report.

**Comment by boegel** [comment]
@valtandor: good to go when Jenkins gives the green light

**Comment by boegel** [comment]
Going in, thanks @valtandor!

**Pull request merged**